core/consensus/grandpa/impl/grandpa_impl.cpp

```diff
@@ -1096,22 +1107,47 @@ namespace kagome::consensus::grandpa {
     ctx->peer_id.emplace(peer_id);
     ctx->commit.emplace(msg);
 
-    auto res = applyJustification(justification.block_info, justification);
-    if (not res.has_value()) {
-      SL_WARN(
-          logger_,
-          "Commit with set_id={} in round={} for block {} has received from {} "
-          "and has not applied: {}",
-          msg.set_id,
-          msg.round,
-          BlockInfo(msg.message.target_number, msg.message.target_hash),
-          peer_id,
-          res.error().message());
+    // Check if commit of already finalized block
+    if (block_tree_->getLastFinalized().number
+        >= justification.block_info.number) {
       return;
```

Comment:
Add log that we received commit that was not applied because it finalizes block less than our best finalized one

Reply:
done